nanofi/include/cxx/Instance.h

```diff
@@ -134,7 +134,7 @@ class Instance {
 
   void transfer(const std::shared_ptr<FlowFileRecord> &ff, const std::shared_ptr<minifi::io::DataStream> &stream = nullptr) {
     std::shared_ptr<core::controller::ControllerServiceProvider> controller_service_provider = nullptr;
-    auto processContext = std::make_shared<core::ProcessContext>(proc_node_, controller_service_provider, no_op_repo_, no_op_repo_, configure_, content_repo_);
+    auto processContext = std::make_shared<core::ProcessContext>(proc_node_, controller_service_provider.get(), no_op_repo_, no_op_repo_, configure_, content_repo_);
```

Comment:
Why no `nullptr` substitution in this file?

Reply:
I do not know. Will replace this with a `nullptr` literal.

Reply:
Updated.